### DISCUSSION TITLE
Welcome to BUCT_COVID-19_Report Discussions!

### DISCUSSION THREAD

**Discussion by W0n9** [W0n9/BUCT_COVID-19_Report#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HccPlus** [comment]
Da Lao New Pee! I'm a student from BUCT, decide to try this[HuaJi].
